src/app/screens/Home/index.jsx

```diff
@@ -138,6 +140,10 @@ function Home() {
                 ),
               }))}
             />
+          ) : (
+            <p className="text-gray-500 dark:text-gray-400">
+              No transactions yet.
```

Comment:
this is also website specific, isn't it? 
maybe we say No transactions on xxx yet?

Reply:
Correct, nice suggestion.
<img width="383" alt="Schermafbeelding 2022-02-09 om 12 06 40" src="https://user-images.githubusercontent.com/12894112/153186183-7fb68317-4226-461a-aedb-14c6db44a92c.png">